app/modules/teams/helpers/teamMembership.ts

```diff
@@ -0,0 +1,18 @@
+import map from 'lodash/map';
+import type { User } from '~/modules/users/users.types';
+
+export async function isTeamMember({ user, teamId }: { user: User, teamId: string }): Promise<boolean> {
```

Comment:
Just want to check that Super admins do not have access to every team. However they should be able to view all teams. But they must be in the team to view, edit, delete and create projects and prompts